USING_SUBSCRIPTIONS.md

```diff
@@ -0,0 +1,102 @@
+# Subscriptions
+
+The FHIR Subscription resource is used to define a push-based subscription from a server to another system. 
+Once a subscription is registered with the server, the server checks every resource that is created or updated, 
+and if the resource matches the given criteria, it sends a message on the defined "channel" so that another system can take an appropriate action.
```

Comment:
channel shouldn't be in quotations. It should either just be channel, or if there is a different word that would more accurately describe this we should use that.